### DISCUSSION TITLE
how to create HA in dispatch

### DISCUSSION THREAD

**Discussion by csetarun2** [Netflix/dispatch#2553] [Q&A] [answered]
I have created aws postgresql rds instance, shall I create 3 instances and in the .env file configure those value and run the docker compose up in all 3 instances, and on top of it shall I use loadbalancer, will it work ? since all vms are going to be use same db will it impact / is there any other approach there..

![image](https://user-images.githubusercontent.com/21236859/192329674-191b8844-4295-496d-9f10-b56a23630ce2.png)

**Comment by Ant0wan** [comment, marked as answer]
@csetarun2 yes it will work.
You can put elb in front of your EC2. Your EC2 need either to run dispatch directly using systemd or running dispatch container image. The only prerequisite is to plug all your dispatch server to the same DB. If you use AWS service I would recommand to use RDS services. :-)
Wish it helped you :nerd_face:

**Reply by csetarun2** [reply]
Thanks @Ant0wan for your reply, Ya I have tried its working :)
